src/test/java/com/github/wasiqb/coteafs/datasource/YmlTest.java

```diff
@@ -0,0 +1,73 @@
+package com.github.wasiqb.coteafs.datasource;
+
+import static com.google.common.truth.Truth.assertWithMessage;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import com.github.wasiqb.coteafs.datasource.data.SampleFile;
+import com.github.wasiqb.coteafs.datasource.data.XmasFifthDay;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * @author Faisal Khatri
+ * @since Aug 29, 2020
+ */
+public class YmlTest {
```

Comment:
You can rename and use existing test `DataSourceTest` as it is also testing `yml` file. No need for a new class.

Reply:
Done